sdk/avalon_sdk/ethereum/proxy/ethereum_work_order.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2020 Intel Corporation
```

Comment:
Can you please add unit test for this class.

Reply:
I will be adding that and enabling all of them(including the 2 existing unit test files) in another PR.